### ISSUE TITLE
TensorFlow - lesson 7

### ISSUE THREAD

**Issue by plouismarie** [udacity/dlnd-issue-reports#255]
I use this code :

def run():
    output = None
    logit_data = [2.0, 1.0, 0.1]
    logits = tf.placeholder(tf.float32)
    
    # TODO: Calculate the softmax of the logits
    softmax = tf.nn.softmax(logit_data)     
    
    with tf.Session() as sess:
        # TODO: Feed in the logit data
        output = sess.run(softmax)

    return output

instead of this code :

def run():
    output = None
    logit_data = [2.0, 1.0, 0.1]
    logits = tf.placeholder(tf.float32)

    softmax = tf.nn.softmax(logits)

    with tf.Session() as sess:
        output = sess.run(softmax, feed_dict={logits: logit_data})

    return output

and it says my answer is the good one

**Comment by Brok-Bucholtz** [comment]
That works, but you also created a neural network without any input.  It's like turning
```
def foo(name):
    print('Hello {}'.format(name))

foo('Emily')
```
into
```
def foo():
    print('Hello Emily')

foo()
```
